apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/filter-table-data.ts

```diff
@@ -1,9 +1,24 @@
+import { IMultilingualText } from '@coscrad/api-interfaces';
 import { isNonEmptyString, isNull, isUndefined } from '@coscrad/validation-constraints';
 
 export type Matchers<T> = {
     [K in keyof T]?: (value: T[K], searchTerm: string) => boolean;
 };
 
+// Break out into separate file and test?
+const isMultilingualText = (text: unknown): text is IMultilingualText => {
```

Comment:
What would happen, for example, if the API exposes (via Redux) a multilingual text property that the index-view designer decided to omit from the table?